src/freertos_drivers/common/DeviceBuffer.cxx

```diff
@@ -34,7 +34,8 @@
 
 #include "DeviceBuffer.hxx"
 
-#ifndef ARDUINO
+#include "openmrn_features.h"
+#if OPENMRN_FEATURE_DEVTAB
```

Comment:
ifdef ?

Reply:
It could be done either way, I modeled this one after other feature flag usages since they are bool in nature. If you prefer ifdef I'll change it over.

My only concern on this change is for the STM32 Arduino port since it should also not include this functionality.

Reply:
@balazsracz looking at the usage of FEATURE_* in the codebase there seems to be a mix of #if and #ifdef. Which would you prefer, I can update this one or send in a PR with mass cleanup for consistency of others.